documentation/staging/content/quickstart/cleanup.md

```diff
@@ -2,22 +2,23 @@
 title: "Clean up"
```

Comment:
@ankedia @rjeberhard - I am still a little worried about accounting for shared operator resources and the multi-operator conflict case:

- Should these cleanup instructions include a step for removing the CRD (assuming this is the only operator in the cluster)?
- Do we need to worry here about the operator uninstall potentially uninstalling another running operator's webhook?
- Or should we simply add a sentence in the beginning that the quick start sample assumes that there are no other operators running in the cluster?